brambling/models.py

```diff
@@ -651,13 +664,13 @@ def remove_from_cart(self, bought_item):
             self.cart_start_time = None
             self.save()
 
-    def mark_cart_paid(self):
+    def mark_cart_paid(self, status):
```

Comment:
Should this method be renamed, since it no longer marks a cart as paid, necessarily?